views/root.ejs

```diff
@@ -1,71 +1,27 @@
 <% title = 'OpenMRS ID - '+user.displayName %>
 <% name = 'root' %>
-<% headline = 'Welcome to the Dashboard, '+user.displayName+'.' %>
+<!-- <% headline = 'Welcome to the OpenMRS Community!" %> -->
```

Comment:
This line appears to be causing an error (Unexpected token ILLEGAL). Since the `<% %>` is compiled by EJS, not the client's browser, HTML comments don't do anything. I'd just remove the line if you don't need it.